lib/ver/CMakeLists.txt

```diff
@@ -12,14 +12,21 @@
 project("ML Ver")
 
 # Prepare values to substitute into the template
-execute_process(COMMAND date "+%Y" OUTPUT_VARIABLE BUILD_YEAR OUTPUT_STRIP_TRAILING_WHITESPACE)
-execute_process(COMMAND id COMMAND awk -F ")" "{ print $1 }" COMMAND awk -F "(" "{ print $2 }" OUTPUT_VARIABLE USER_NAME
-  OUTPUT_STRIP_TRAILING_WHITESPACE)
-execute_process(COMMAND awk -F= "/elasticsearchVersion/ {gsub(/-.*/,\"\"); print $2}"
-  $ENV{CPP_SRC_HOME}/gradle.properties OUTPUT_VARIABLE PRODUCT_VERSION OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-if(${VERSION_QUALIFIER})
-  set(PRODUCT_VERSION "${PRODUCT_VERSION}-${VERSION_QUALIFIER}")
+string(TIMESTAMP BUILD_YEAR "%Y")
+
+if(MSVC)
```

Comment:
nit: Technically this is a feature of Windows, not Visual Studio, so maybe we should check for Windows here. (Obviously it would only matter if we decided to build with something else on Windows in the future.)

Reply:
I'll change this to `WIN32`